### PR TITLE
chore(e2e): skip flaky e2e for vue and angular platforms

### PR DESCRIPTION
**Pull request by ioanabrooks**
#### Description of changes

The verify user e2e test has been failing for [angular](https://github.com/aws-amplify/amplify-ui/actions/runs/5685875411/job/15412098132?pr=4290) and [vue](https://github.com/aws-amplify/amplify-ui/actions/runs/5696499143/job/15441917416?pr=4295) platforms, this change temporarily skips it to unblock PRs. Ongoing work to follow in #4295. 

Also skip liveness tests for now to unblock pipeline.

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/e2e/features/ui/components/authenticator/verify-user.feature [hunk 1]
```diff
@@ -49,7 +49,7 @@ Feature: Verify User
     Then I see "Sign out"
     And I click the "Sign out" button
 
-  @angular @react @vue @react-nativee
+  @todo-angular @react @todo-vue @todo-react-native
```
Comment (ioanabrooks) [inline packages/e2e/features/ui/components/authenticator/verify-user.feature:52]: * CI is set up to ignore @todo-framework tests
* react native tests were not running for this suite, I have a follow-up asana ticket to address this (tldr, this feature reuses sign-in example app which is not mocked, due to how Detox handles mocking would need a separate app)

**Review by calebpollman — APPROVED** [review]

**Review by reesscot — DISMISSED** [review]

File: packages/e2e/features/ui/components/authenticator/verify-user.feature [hunk 2]
```diff
@@ -49,7 +49,7 @@ Feature: Verify User
     Then I see "Sign out"
     And I click the "Sign out" button
 
-  @angular @react @vue @react-nativee
```
[the remark this answers is not in the document]
Comment (ioanabrooks) [reply packages/e2e/features/ui/components/authenticator/verify-user.feature:52]: https://github.com/aws-amplify/amplify-ui/pull/4298#discussion_r1278173720

**Review by calebpollman — COMMENTED** [review]
Kind of wondering if we should have separate PRs since we are tracking two different issues here?

File: packages/e2e/features/ui/components/authenticator/verify-user.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (calebpollman) [reply packages/e2e/features/ui/components/authenticator/verify-user.feature:52]: Still curious abt the typo for my own understanding

**Comment by ioanabrooks** [comment]
> Kind of wondering if we should have separate PRs since we are tracking two different issues here?

I'm thinking the issue is the pipeline being blocked. Actual fixes will be addressed in separate PRs. We can't separate these without admin merging anyway/

File: packages/e2e/features/ui/components/authenticator/verify-user.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (ioanabrooks) [reply packages/e2e/features/ui/components/authenticator/verify-user.feature:52]: Not a typo, wasn't meant to be running.

**Review by reesscot — APPROVED** [review]

**Review by zchenwei — APPROVED** [review]

File: packages/e2e/features/ui/components/authenticator/verify-user.feature (hunk 2 above)
[the remark this answers is not in the document]
Comment (calebpollman) [reply packages/e2e/features/ui/components/authenticator/verify-user.feature:52]: Wait, we don't use `@react-native`? That's what i've seen in all the other test files

**Pull request merged**